gradle.properties

```diff
@@ -2,7 +2,7 @@ kotlin.code.style=official
 kotlin.jvm.target=11
 
 GROUP=io.github.kevincianfarini.monarch
-VERSION_NAME=0.2.2
+VERSION_NAME=0.3.0-SNAPSHOT
```

Comment:
Going to push a snapshot release and dogfood it to see if there's anything weird going on with static linking.